app/src/main/java/com/simple/demotest/MainAction.java

```diff
@@ -0,0 +1,35 @@
+package com.simple.demotest;
+
+import android.content.Context;
+import android.util.Log;
+import android.widget.Toast;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.UnusedImportsCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/demotest/MainAction.java L5

Unused import: android.widget.Toast. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*